packages/react-app-revamp/config/wagmi/custom-chains/degenChain.ts

```diff
@@ -0,0 +1,24 @@
+import { Chain } from "@rainbow-me/rainbowkit";
+
+export const nautilusChain: Chain = {
```

Comment:
Make sure to update the chain name here!

Reply:
updated!